samples/net/google_iot_mqtt/src/globalsign.inc

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
We have a way to automatically generate the .inc file, see for example `samples/net/sockets/echo_server/CMakeLists.txt` how to do it. That way it is much easier to replace the cert when needed.

Reply:
so basically if i have a .der encoded file it'll generate the rest.  interesting

Reply:
Did you investigated how the .inc file could be generated automatically? This is not a blocker for this PR but might be good to have in the future.